libvast/CMakeLists.txt

```diff
@@ -175,18 +175,18 @@ if (NOT VAST_ENABLE_BUNDLED_CAF)
   # Try to find the required CAF components first...
   provide_find_module(CAF)
   find_package(
-    CAF 0.17.6 EXACT
+    CAF 0.18.6 EXACT
```

Comment:
This must also be adapted in the docs (there's a table listing versions in the build from source section), and it also should have its own changelog entry that lists all the user-facing changes, e.g.,

> Building VAST now requires CAF 0.18.6. VAST supports setting advanced options for CAF directly in its configuration file under the `caf` section. If you were using any of these, compare them against the bundled `vast.yaml.example` file to see if you need to make any changes. The change has (mostly positive) [performance and stability implications](https://www.actor-framework.org/blog/2021-01/benchmarking-0.18/) throughout VAST, especially in high-load scenarios.

Similarly, the `vast.yaml.example` file should be updated to contain all the new CAF options rather than the old ones.

Reply:
I've written a follow-up story for the vast.yaml.example file so we can unblock this.